### ISSUE TITLE
`app.error` writes `[log]` instead of `[err]`

### ISSUE THREAD

**Issue by OlivierZal** [athombv/apps-sdk-issues#269]
`app.error` writes `[log]` instead of `[err]`

**Comment by WeeJeWel** [comment]
This issue should have been resolved somewhere in September. Which Homey Pro version are you using?

**Comment by OlivierZal** [comment]
```
HomeyVersion | "8.0.8"
homeyModelId | "homey4d"
homeyModelName | "Homey Pro (Early 2019)"
```

**Comment by RobinBol** [comment]
@OlivierZal this will be fixed in Homey v8.1.1, will be released in the coming weeks.

**Comment by RobinBol** [comment]
Homey v8.1.1 is stable.
